components/var_dumper.rst

```diff
@@ -326,11 +326,11 @@ Example::
             // the expected dump contents don't have the default VarDumper structure
             // because of the custom casters and flags used in the test
             $expectedDump = <<<EOTXT
-    [
-      123,
-      "foo",
-    ]
-    EOTXT;
+            [
```

Comment:
I thought the same in the past, but ended up with a whitelist entry.

WDYT @javiereguiluz